rpcs3/Emu/RSX/VK/VKDraw.cpp

```diff
@@ -161,11 +161,12 @@ void VKGSRender::load_texture_env()
 			(update_framebuffer_sourced && fs_sampler_state[i]->upload_context == rsx::texture_upload_context::framebuffer_storage))
 		{
 			auto sampler_state = static_cast<vk::texture_cache::sampled_image_descriptor*>(fs_sampler_state[i].get());
+			auto texture = rsx::method_registers.fragment_textures[i];
 
-			if (rsx::method_registers.fragment_textures[i].enabled())
+			if (texture.enabled())
```

Comment:
@Nekotekina whether in function **load_texture_env()** to overwrite the condition on
`if (texture.enabled() && sampler_state->validate())` ?